### PR TITLE
actually track $msefl now and also properly track $iuscp

### PR DESCRIPTION
**Pull request by VVelox**
Found this in https://github.com/librenms/librenms/pull/6173 .

### CODE REVIEW

**Comment by VVelox** [comment]
Much thanks to @florianbeer for the suggestions. :)

File: snmp/postfixdetailed [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/local/bin/perl
```
Comment (laf) [inline snmp/postfixdetailed]: What's up with using env here?

File: snmp/postfixdetailed [hunk 2]
```diff
@@ -7,7 +7,7 @@
 my $cache='/var/cache/postfixdetailed';
 
 # the location of pflogsumm
-my $pflogsumm='/usr/bin/env pflogsumm';
+my $pflogsumm='/usr/local/bin/pflogsumm';
```
Comment (laf) [inline snmp/postfixdetailed]: Same again

**Comment by VVelox** [comment]
@florianbeer Whoops. Yeah. Copied and paste of it from the system I was testing on did that. Fixed now.

@laf Fixed.

**Comment by VVelox** [comment]
Also, no, this does not break any existing systems as this appends the variable to the end. As to the other changes, those just make sure stuff is properly counted.

**Comment by laf** [comment]
Why not just have the small change done in florians PR?

**Comment by VVelox** [comment]
Could not make the changes, it would of broken stuff because of the positioning of $msefl during the print, and I also found another small bug as well.

**Comment by laf** [comment]
But it would be better to just collaborate with him on his PR :)

**Pull request closed without merging**